Dockerfile

```diff
@@ -10,7 +10,12 @@
 
 FROM alpine:3.10.2
 
-ENV HOME=/home/theia
+# odo and oc versions have to match the ones defined in https://github.com/redhat-developer/vscode-openshift-tools/blob/master/src/tools.json
+ENV GLIBC_VERSION="2.30-r0" \
+    ODO_VERSION="v1.1.0" \
```

Comment:
since this image is for openshift-connector plugin 0.1.2 version, then ODO should be 1.0.1 version, since plugin is configured to work with that one